packages/config/src/forkConfig/types.ts

```diff
@@ -40,7 +40,7 @@ export type ForkConfig = {
   /** Get post-deneb SSZ types by hard-fork*/
   getPostDenebForkTypes(slot: Slot): SSZTypesFor<ForkPostDeneb>;
   /** Get max blobs per block by hard-fork */
-  getMaxBlobsPerBlock(fork: ForkName): number;
+  getMaxBlobsPerBlock(epoch: Epoch): number;
   /** Get max request blob sidecars by hard-fork */
   getMaxRequestBlobSidecars(fork: ForkName): number;
```

Comment:
might also wanna update this but it might be removed soon and we might be fine just calculating it where we need it

Reply:
maybe we get rid of this function altogether and always calculate the value as proposed in https://github.com/ethereum/consensus-specs/pull/4322

Reply:
> maybe we get rid of this function altogether and always calculate the value as proposed in [ethereum/consensus-specs#4322](https://github.com/ethereum/consensus-specs/pull/4322)

This change will likely be in devnet 1. Will follow up in another PR

Reply:
> This change will likely be in devnet 1. Will follow up in another PR

In devnet-0 the req/resp limits will be wrong then, we should consider doing this already for that devnet, can do it in a separate PR though to keep diff minimal here and focus on blob schedule changes

Reply:
> might also wanna update `getMaxRequestBlobSidecars` but it might be removed soon and we might be fine just calculating it where we need it

As pointed out here https://github.com/ChainSafe/lodestar/pull/7729#issuecomment-2896820306 , devnet-0 is still using `MAX_BLOBS_PER_BLOCK_ELECTRA` and `MAX_BLOBS_PER_BLOCK` for electra and deneb.

Can hold off making this change until devnet-1

Reply:
But still, we need to update those values for fulu and later, otherwise we keep using the electra values, I don't think it's a big deal though